packages/connect/src/utils/assets.ts

```diff
@@ -9,13 +9,37 @@ if (global && typeof global.fetch !== 'function') {
     global.fetch = fetch;
 }
 
-export function httpRequest(url: string, type: 'text'): Promise<string>;
-export function httpRequest(url: string, type: 'binary'): Promise<ArrayBuffer>;
-export function httpRequest(url: string, type: 'json'): Promise<Record<string, any>>;
-export function httpRequest(url: any, type: any) {
-    const asset = getAssetByUrl(url);
+export function httpRequest(
+    url: string,
+    type: 'text',
+    options?: RequestInit,
+    skipLocalForceDownload?: boolean,
```

Comment:
side note: how is this change related with https://github.com/trezor/trezor-suite/issues/13271